sess.go

```diff
@@ -92,12 +92,20 @@ func (s *session) Serve() error {
 		return err
 	}
 
+	s.initialized = true
+
 	// handle authentication.
-	err = s.Write(authOKMsg())
+	// TODO: replace with an actual pre-configured authenticator
+	a := &noPasswordAuthenticator{}
```

Comment:
the authenticator should be defined on Server instance, not on each and every session creation (from previous review cycle)

Reply:
@aviaoh I thought to move it when we have a real authenticator. Will move now.